bftengine/src/bcstatetransfer/InMemoryDataStore.hpp

```diff
@@ -171,8 +171,8 @@ class InMemoryDataStore : public DataStore {
 
   std::atomic_bool fetching{false};
 
-  CheckpointDesc checkpointBeingFetched;
-  // none if checkpointBeingFetched.checkpointNum == 0
+  CheckpointDesc checkpointBeingFetched_;
+  // none if checkpointBeingFetched_.checkpointNum == 0
```

Comment:
Suggest to refactor the comment and put above, not bellow.
```
// No checkpoint is being fetched if checkpointBeingFetched_.checkpointNum is 0
```

Reply:
Thanks, refactored.